content/oc/_index.md

```diff
@@ -162,4 +162,79 @@ done
 
 ```
 oc logs <pod> --all-containers
-```
\ No newline at end of file
+```
+
+# Create Kubeconfig out of credentials / Merge multiple Kubeconfigs
```

Comment:
Can you split this into 2 different sections? how to create a kubeconfig out of credentials and merge multiple kubeconfigs (I believe it is confusing to have those mixed)